Client/sdk/game/CCarEnterExit.h

```diff
@@ -23,4 +23,6 @@ class CCarEnterExit
                                             bool bCheckIfRoomToGetIn) = 0;
     virtual int  ComputeTargetDoorToExit(CPed* pPed, CVehicle* pVehicle) = 0;
     virtual bool IsRoomForPedToLeaveCar(CVehicle* pVehicle, int iDoor, CVector* pUnknown = 0) = 0;
+
+    virtual const void GetPositionToOpenCarDoor(CVector& position, CVehicle* vehicle, unsigned int door) noexcept = 0;
```

Comment:
`unsigned int door` -> `uint32_t door`

Reply:
`std::uint32_t`*

Reply:
What's wrong with uint32_t? std::uint32_t and uint32_t are the same.

Reply:
> What's wrong with uint32_t? std::uint32_t and uint32_t are the same.

We use C++ so lets stick to C++ headers. Thus, `std::` is mandatory. `cstdint` will include `uint` in the `std` namespace but may or may not include them in the global namespace and vice versa